index.js

```diff
@@ -1,5 +1,7 @@
 /*! standard. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
-const Linter = require('standard-engine').linter
-const opts = require('./options')
+import engine from 'standard-engine'
```

Comment:
Would be nice to convert `standard-engine` to ESM also, and then be able to use `import { Linter } from 'standard-engine'`

Reply:
yea... but what should come first? the chicken or the egg? standard/standard-engine#251 (comment)

Reply:
Yeah, I don't have an opinion on which comes first 👍

Reply:
Indeed. :+1:
Actually, this is the reason for this PR (to have a smoother transition to ESM across the organization).
There is an open issue in `standard-engine`: https://github.com/standard/standard-engine/issues/251

Reply:
I like the naming convention doe. linter is a class so it should be exported/renamed to `Linter` from `linter`

Reply:
There is already a PR for that change in `standard-engine`: https://github.com/standard/standard-engine/pull/253